cgi/CRMS.pm

```diff
@@ -1436,19 +1436,23 @@ sub LoadQueueForProject {
   my $project_name = $self->GetProjectRef($project)->name;
   my $sql = 'SELECT COUNT(*) FROM queue WHERE project=?';
   my $queueSize = $self->SimpleSqlGet($sql, $project);
-  my $targetQueueSize = $self->GetProjectRef($project)->queue_size();
+  my $project_ref = $self->GetProjectRef($project);
+  my $targetQueueSize = $project_ref->queue_size();
   my $needed = $targetQueueSize - $queueSize;
   $self->ReportMsg("Project $project_name: $queueSize volumes -- need $needed");
   return if $needed <= 0;
   my $count = 0;
+  my @orders = ('c.time DESC');
```

Comment:
I would probably have done it like:

```perl
# value(s) for ORDER BY: include & prioritize "project order" if defined
my $orders = join(' ', grep { defined $_ } ($project_ref->queue_order, 'c.time DESC'));
```

... but there is of course more than one way to do it.

Reply:
I have a GH issue to redefine these optional methods as returning an arrayref, default empty. That way it's just an array merger without having to deal with `undef`. Out of scope for a TTO, but yeah definitely could do better.